cli/cli/commands/lint/lint.go

```diff
@@ -40,6 +46,8 @@ const (
 	presentWorkingDirectory = "."
 	versionArg              = "version"
 
+	mainDotStarFilename = "main.star"
```

Comment:
Let's keep it `main.star` I think it's valid and then we can discuss with the team about the `--main-file` flag, if we should rename it to avoid confusions.

Reply:
This makes sense! We can iteratively add it later.